build-tools/packages/build-tools/package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "@fluidframework/build-tools",
-	"version": "0.28.0",
+	"version": "0.29.0",
```

```suggestion
	"version": "0.28.0",
```

No need for this change.